docs/key_features.md

```diff
@@ -0,0 +1,28 @@
+# Key Features of `scores`
+
+## Data Handling
+
+- Works with labelled, n-dimensional data (e.g., geospatial, vertical and temporal dimensions) for both point-based and gridded data. `scores` can effectively handle the dimensionality, data size and data structures commonly used for:
+  - gridded Earth system data (e.g., numerical weather prediction models)
+  - tabular, point, latitude/longitude or site-based data (e.g., forecasts for specific locations).
+- Handles missing data, masking of data and weighting of results.
+- Supports [xarray](https://xarray.dev/) datatypes, and works with [NetCDF4](https://doi.org/10.5065/D6H70CW6), [HDF5](https://github.com/HDFGroup/hdf5), [Zarr](https://zarr.dev) and [GRIB](https://codes.wmo.int/grib2) data sources among others. 
+
```

Comment:
Should/could a dot point be added at the end of "Data Handling" section, along the lines of:
- [Some metrics](https://scores.readthedocs.io/en/228-documentation-testing-branch/included.html#pandas) work with [pandas](https://pandas.pydata.org/).

(I don't know if this actually falls under data handling or not).

If so, would remove the final dot point in the "Compatibility" section (which is about expanding support for pandas). If leaving the pandas dot point in the "compatibility" section, it might be worth changing the existing pandas dot point to the one suggested in this comment.

Reply:
Tennessee says leave this as is for now.